apps/aecore/test/aec_test_utils.erl

```diff
@@ -135,46 +138,58 @@ unmock_block_target_validation() ->
     meck:unload(aec_target).
 
 
-genesis_block() ->
-    aec_block_genesis:genesis_block(?PRESET_ACCOUNTS).
+genesis_block_with_state() ->
+    aec_block_genesis:genesis_block_with_state(?PRESET_ACCOUNTS).
+
+genesis_block_and_state() ->
+    {ok, B, S} = genesis_block_with_state(),
+    {B, S}.
 
 %% Generic blockchain with only coinbase transactions
 gen_block_chain(Length) ->
     gen_block_chain(Length, ?PRESET_ACCOUNTS).
-  
+
+gen_block_chain_without_state(Length) ->
```

Comment:
This is a missnomer. The block chain is generated with the state. It just doesn't return it.

It would be better to pick a new name, or to rename the internal functions.